src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,27 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
```

Comment:
```suggestion
title: RoboCup Network Setup
```

Reply:
I was thinking we could have more than just this guide on this page :shrug:.